samples/sensor/bme280/boards/intel_adl_crb.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c0 {
+        bme280@76 {
+                compatible = "bosch,bme280";
+                reg = <0x76>;
+		status = "okay";
+        };
+};
```

Comment:
Is this sensor on the board? If it is, this node should be defined in the board dts instead.

Reply:
It is not on board sensor, we have connected it to the board and tested it. So, we are not included in board dts.